cmd/kube-scheduler/app/server.go

```diff
@@ -159,11 +159,8 @@ func Run(cc schedulerserverconfig.CompletedConfig, stopCh <-chan struct{}, regis
 	// To help debugging, immediately log version
 	klog.V(1).Infof("Starting Kubernetes Scheduler version %+v", version.Get())
 
-	registry := plugins.NewDefaultRegistry()
-	for _, option := range registryOptions {
-		if err := option(registry); err != nil {
-			return err
-		}
+	if len(registryOptions) != 0 {
```

Comment:
Instead of totally disabling it, does a warning giving the limitations sound better? If so, does it resolve @hex108 's concern?